examples/validate-fee/sample-ui/src/app/polls/create/form.tsx

```diff
@@ -0,0 +1,203 @@
+'use client';
+
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { z } from 'zod';
+import { useEffect } from 'react';
+import { toast } from 'react-toastify';
+
+import { ButtonLink } from '../../../components/Button/ButtonLink/component.tsx';
+import { useWalletProvider } from '../../../providers/index.ts';
+import { PollSchema } from '../../../schemas/index.ts';
+import { buildRegisterField } from '../../../utils/forms.ts';
+import { Button, Card, Input } from '../../../components/index.ts';
+import { useToastAction } from '../../../utils/use_toast_action.ts';
```

Comment:
it's better to use aliases like `@schemas/`, `@components/` or `@utils/` than `../../../`

Reply:
Deleted sample-ui